packages/good-design/src/apps/goodid/wizards/RedtentWizard.tsx

```diff
@@ -263,10 +282,17 @@ export const RedtentWizard: React.FC<RedTentProps> = (props: RedTentProps) => {
   const [error, setError] = useState<Error | undefined>(undefined);
   const { videoInstructStyles } = props;
   const dontShowAgainKey = "goodid_noOffersModalAgain";
+  const { track } = useSendAnalytics();
   // inject show modal on callbacks exceptions
 
   const modalOnDone: RedTentProps["onDone"] = async errorOnDone => {
     try {
+      if (!isBoolean(errorOnDone)) {
+        track("offer_video_error", { error: errorOnDone });
+      }
+
+      track("offer_video_success");
```

Comment:
if this is tracking video success, where is the general offer success?

Reply:
I dont understand, what general offer success?

there is no general offer flow. the mvp offer we have for redtent is not generic enough to be re-used by other/new offers

Reply:
once the video is uploaded there is only thank you screen and you will be navigated to the claim-flow.

offer_video_success is the 'general offer success' the way I look at it.

I don't know if a additional 'offer_success' event here: https://github.com/GoodDollar/GoodWeb3-Mono/blob/b8161414cf69135196cc7b5dfc4bf912e74c2dcc/packages/good-design/src/apps/goodid/wizards/RedtentWizard.tsx#L243

will give much value or insight